index.js

```diff
@@ -449,11 +449,11 @@ module.exports = {
         'p--whats-the-crime-reference-number': {
             $schema: 'http://json-schema.org/draft-07/schema#',
             type: 'object',
-            title: "What's the crime reference number?",
```

Comment:
can we single-quote this (and any others to make that consistent). I'm surprised the linter didn't catch these.